flake.nix

```diff
@@ -188,31 +201,26 @@
           # `devShells`)
           treefmt.config = {
             projectRootFile = "flake.nix";
-            programs = { nixpkgs-fmt.enable = true; }
-              # FIXME Ormolu segfaults on `aarch64-darwin` so it should be
-              # disabled everywhere (i.e. just omit it from `treefmt.config`).
-              #
-              # See https://github.com/plow-technologies/inferno/issues/10
-              // lib.optionalAttrs (system != "aarch64-darwin")
-              {
-                ormolu = {
-                  enable = true;
-                  package =
-                    let
-                      # Using `hackage-package` will prevent building `ormolu`
-                      # from interfering with the build plan (incl. incompatible
-                      # compiler versions)
-                      o = pkgs.haskell-nix.hackage-package {
-                        name = "ormolu";
-                        version = "0.5.0.1";
-                        compiler-nix-name = defaultCompiler;
-                        configureArgs = "--disable-benchmarks --disable-tests";
-                      };
-                    in
-                    o.getComponent "exe:ormolu";
-                  ghcOpts = [ "TypeApplications" ];
-                };
+            programs = {
+              nixpkgs-fmt.enable = true;
+              ormolu = {
+                enable = true;
+                package =
+                  let
+                    # Using `hackage-package` will prevent building `ormolu`
+                    # from interfering with the build plan (incl. incompatible
+                    # compiler versions)
+                    o = pkgs.haskell-nix.hackage-package {
```

Comment:
I've switched the formatter to Fourmolu, for several reasons:
- We use it elsewhere
- It's better than Ormolu
- Even staying with Ormolu will require a large reformatting of the Haskell codebase, so we might as well switch to Fourmolu

I've disabled the CI check for formatting for now. Once the upgrade is in and confirmed working everywhere, I will re-enable it in a re-formatting PR